src/guide/jbs-jdk-bug-system.md

```diff
@@ -736,7 +736,7 @@ Examples:  If a bug fix only corrects a change in the build system, then add the
   <tr>
     <td class="dictionary">[[regression_]{.jbs-label}*(ID)*]{#regression_id}</td>
     <td class="dictionary">
-      Used to identify the fix that caused the regression, where known.
+      **Deprecated.** Was used to identify the fix that caused the regression, where known. Has been replaced by the use of the [caused by]{.jbs-value} link.
```

Comment:
Maybe "**Formerly** used to identify...".

Also either "**This label** has been replaced by the use..." or "**Replaced** by the use..."